wdl-analysis/src/handlers/goto_definition.rs

```diff
@@ -420,8 +420,40 @@ impl GotoDefinitionHandler {
             .unwrap_or(crate::types::Type::Union);
 
         if let Some(struct_ty) = target_type.as_struct() {
+            debug!("found a struct type: {}", struct_ty);
             let struct_def = analysis_doc
-                .struct_by_name(struct_ty.name())
+                .structs()
+                .find(|(_, s)| {
+                    // When structs imported and aliased to a different name,they are checked
+                    // using their Type
+                    //
+                    // Eg:
+                    // `person.wdl`:
+                    //
+                    // sturct Person {
+                    //  Sting name
+                    // }
+                    //
+                    // `main.wdl`:
+                    //
+                    // import "person.wdl" as lib alias Person as person
+                    //
+                    // workflow main {
+                    //  peson a = person {
+                    //      name = "wdl"
+                    //  }
+                    //
+                    //  a.name
+                    //    ^^^^ this has a struct name as person but Type is `Person`
```

Comment:
Got aliased import navigation working, had to ref [SPEC](https://github.com/openwdl/wdl/blob/wdl-1.2/SPEC.md#importing-and-aliasing-structs) a couple of times

also should i remove this ? @claymcleod